bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonNetworkHandler.java

```diff
@@ -129,6 +146,12 @@ public void initialize() {
     public void dispose() {
         logger.debug("Shutting down Insteon bridge");
 
+        ScheduledFuture<?> driverInitializedJob = this.driverInitializedJob;
+        if (driverInitializedJob != null) {
+            driverInitializedJob.cancel(false);
```

Comment:
You should rather use true as parameter.

Reply:
fixed